molecule/cluster-upgrade/prepare.yml

```diff
@@ -25,7 +25,7 @@
       delegate_to: "{{ groups['controllers'][0] }}"
       kubernetes.core.k8s_info:
         kind: Node
-        name: "{{ inventory_hostname_short }}"
```

Comment:
This is used for CI and made the tests fail, we can keep this one as it's just a readiness condition check.

Reply:
Sure. I've pushed a revert which can be squashed down.